File: prak_lat1 php/Ram.php
<?php
class Ram{

	// Attributes
	private $capacity;
	private $price;

	// Constructor
	public function __construct($capacity, $price){
		$this->capacity = $capacity;
		$this->price = $price;
	}

	// Set & Get Methods
	public function setCapacity($capacity){
		$this->capacity = $capacity;
	}

	public function setPrice($price){
		$this->price = $price;
	}

	public function getCapacity(){
		return $this->capacity;
	}

	public function getPrice(){
		return $this->price;
	}
}
?>

File: prak_lat1 python/Ram.py
class Ram:

	# Constructor
	def __init__(self, capacity, price):
		self.capacity = capacity
		self.price = price

	# Set & Get methods
	def set_capacity(self, capacity):
		self.capacity = capacity

	def set_price(self, price):
		self.price = price

	def get_capacity(self):
		return self.capacity

	def get_price(self):
		return self.price